frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-resource.tsx

```diff
@@ -127,6 +138,21 @@ export const VMTemplateDetailsList: React.FC<VMTemplateResourceListProps> = ({
         </EditButton>
       </VMDetailsItem>
 
+      <VMDetailsItem
+        title={DEDICATED_RESOURCES}
+        idValue={prefixedID(id, 'dedicated-resources')}
+        canEdit
+        onEditClick={() => setDedicatedResourcesModalOpen(true)}
+        editButtonId={prefixedID(id, 'dedicated-resources-edit')}
+      >
+        <DedicatedResourcesModal
+          vmLikeEntity={template}
+          isOpen={isDedicatedResourcesModalOpen}
+          setOpen={setDedicatedResourcesModalOpen}
```

Comment:
as metioned above, it would be better to pass `onClose` callback
```
const onClose = React.useCallback(() => setDedicatedResourcesModalOpen(false), []);
...
<DedicatedResourcesModal
          vmLikeEntity={template}
          isOpen={isDedicatedResourcesModalOpen}
          onCLose={onClose}



```